test/parse.jl

```diff
@@ -1056,3 +1056,5 @@ macro make_f21054(T)
 end
 @eval @make_f21054 $Array
 @test isa(f21054, Function)
+g21054(>:) = >:2
```

Comment:
Well, this worked before, and the issue is just lowering code that only handles an argument count of 2. No reason the same code shouldn't be able to handle any number of arguments.